app/Providers/ErrorReportingProvider.php

```diff
@@ -154,7 +154,7 @@ public function isReportingEnabled(): bool
     public function handleError($level, $message, $file = '', $line = 0, $context = []): bool
     {
         // report errors if they are allowed
-        if ($this->errorReportingLevel & $level) {
+        if (error_reporting() & $level && $this->errorReportingLevel & $level) {
```

Comment:
If we want to use this, we will have to look about changing: https://github.com/librenms/librenms/blob/master/LibreNMS/Util/Debug.php#L121

Reply:
Hhmmmmm I think this is the wrong fix for it actually.